### PR TITLE
DSP-18644: Update README

### PR DESCRIPTION
**Pull request by justinchuch**

### CODE REVIEW

File: graph-migration/README.md [hunk 1]
```diff
@@ -5,30 +5,41 @@
 Graph Schema Migration tool creates legacy graph frame. Based on legacy graph frame schema, the tool generates native graph schema.
 
 There are two types of schema, cql and gremlin formats. Cql format schema is the CQL script to create graph keyspace and
-tables. Gremlin format is native graph schema.
-Run the following command to generate native graph schema
+tables. Gremlin format is groovy script to create native graph schema under Gremlin console.
```
Comment (rocco408) [inline graph-migration/README.md]: or Studio correct?

File: graph-migration/README.md [hunk 2]
```diff
@@ -5,30 +5,41 @@
...
+
+Run the following command to generate native graph schema with DSE server is up:
```
Comment (rocco408) [inline graph-migration/README.md]: "**when** the DSE server is up"

File: graph-migration/README.md [hunk 3]
```diff
@@ -5,30 +5,41 @@
...
+```
+dse graph-migrate-schema [-cql | -gremlin] <legacy_graph> <native_graph>
 ```
-dse graph-migrate-schema -cql|-gremlin <legacy_graph> <native_graph>
+
+- `-cql`: specifies the output as cql format.
+- `-gremlin`: specifies the output as gremlin format.
```
Comment (rocco408) [inline graph-migration/README.md:17]: Should we also explain what legacy_graph and native_graph are? Something as simple as:
```
<legacy_graph>: the name of the existing legacy graph you which to migrate
<native_graph>: the name of the new native graph you would like to create
```

File: graph-migration/README.md [hunk 4]
```diff
@@ -5,30 +5,41 @@
...
+Please note that the following are NOT supported:
+- Meta properties
+- Multi properties
+- Indexes
+
+All properties are single. Multi properties are migrated as a single property. Meta properties, Indexes, and MVs are
 dropped. If custom handling of meta properties and multi properties, renaming properties and dropping properties are
-needed, user should modify the generated native graph schema. The native graph schema can be created through cqlsh or
-gremlin.
+needed, users should modify the generated native graph schema.
```
Comment (rocco408) [inline graph-migration/README.md]: I would reword to something like:

> If custom handling of meta and multi properties is needed, users should modify the generated native graph schema accordingly, some properties may need to be renamed or dropped.

Side note: we really need some examples for dealing with meta and multi properties. Not something we necessarily need in this PR, but something we need to add at some point soon.

File: graph-migration/README.md [hunk 5]
```diff
@@ -5,30 +5,41 @@
...
 
-Then pass generated schema to cqlsh or gremlin-console depends on the type
+To create the native graph schema, pass the generated schema to cqlsh or gremlin-console depends on the type
```
Comment (rocco408) [inline graph-migration/README.md]: replace "depends" with "depending"

File: graph-migration/README.md [hunk 6]
```diff
@@ -5,30 +5,41 @@
...
 dse gremlin-console -e native_schema.groovy
 ```
 
 ## Migrate Data
 
-Provided script is just an example of the data migration process.
-It should work as is, if new native schema was generated with previous step and was applied without 
-modifications. 
+The provided script is just an example of the data migration process.
+It should work with the native graph schema generated in the previous step, assuming there is no modification to it.
```
Comment (rocco408) [inline graph-migration/README.md]: I assume the generated script would also work after being modified, provided the modification are correct? I'm actually not sure here, more of a question than a statement :)

**Review by rocco408 — APPROVED** [review]
Looks good overall I think, just minor suggestions to polish it off.

File: graph-migration/README.md (hunk 1 above)
Comment (justinchuch) [reply graph-migration/README.md]: agree. Thanks!

File: graph-migration/README.md (hunk 3 above)
Comment (justinchuch) [reply graph-migration/README.md:17]: Good point! Added!

File: graph-migration/README.md (hunk 4 above)
Comment (justinchuch) [reply graph-migration/README.md]: True, I will leave this comment open.

File: graph-migration/README.md (hunk 5 above)
Comment (justinchuch) [reply graph-migration/README.md]: 👍

File: graph-migration/README.md (hunk 6 above)
Comment (justinchuch) [reply graph-migration/README.md]: I made the changes based on the original paragraph. I guess what we try to say here is that the data migration part should work if there is no modification made to the script after it is generated. Otherwise, the data migration would not work because of the modification.

File: graph-migration/README.md (hunk 2 above)
Comment (justinchuch) [reply graph-migration/README.md]: Yup!

File: graph-migration/README.md (hunk 6 above)
Comment (artem-aliev) [reply graph-migration/README.md]: I prefer to state here, something like:
if you modify generated schema, please modify the migration script accordingly. 
And there are some example later, like getLegacyPropertyName() could be overridden for name changes.

File: graph-migration/README.md [hunk 7]
```diff
@@ -40,20 +54,20 @@ and then submit to Spark:
...
-Then the tool enumerate native edge labels, extract (inLabel, edgeLabel, outLabel) triplet from it, select proper rows
-from legacy DGF edges, convert edge ids into native format and write data to corresponded table. See `migrateEdges()` method.
-`handleMultiAndMetaProperies()` method is expected to be overridden to properly migrate multi and meta properties
-    by default it just drops all  metadata and select the first of multi properties.                                  
-`getLegacyPropertyName()` should be overridden if property names was manually changed in the schema generate by the 
-   `graph-migrate-schema`
-   
- 
\ No newline at end of file
+The migration tool uses spark-cassandra-connector to directly write data to DSE-DB.
```
Comment (artem-aliev) [inline graph-migration/README.md]: #2 introduced new update API usage instead of direct used of spark-cassandra-connector

Comment (artem-aliev) [reply graph-migration/README.md]: so you can remove this sentence

File: graph-migration/README.md (hunk 6 above)
Comment (justinchuch) [reply graph-migration/README.md]: Sure, took in your suggestion. 😄

File: graph-migration/README.md (hunk 7 above)
Comment (justinchuch) [reply graph-migration/README.md]: Okay, updated accordingly.
